tt_eager/tt_dnn/op_library/prod/kernels/prod_hw.cpp

```diff
@@ -0,0 +1,62 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include <cstdint>
+
+#include "compute_kernel_api/eltwise_binary.h"
+#include "compute_kernel_api/reduce.h"
+#include "compute_kernel_api/tile_move_copy.h"
+
+ALWI void ACQ() { acquire_dst(tt::DstMode::Half); }
+ALWI void REL() { release_dst(tt::DstMode::Half); }
```

Comment:
Switch to new APIs:
- tile_regs_commit, tile_regs_release
- tile_regs_acquire, tile_regs_wait

Docs: tt_metal/include/compute_kernel_api/reg_api.h